mne/viz/_brain/_brain.py

```diff
@@ -3689,9 +3687,12 @@ def _to_borders(self, label, hemi, borders, restrict_idx=None):
             show[keep_idx] = 1
             label *= show
 
+    @deprecated('enable_depth_peeling is deprecated and will be '
+                'removed in 1.1')
     def enable_depth_peeling(self):
-        """Enable depth peeling."""
-        self._renderer.enable_depth_peeling()
+        """Enable depth peeling.
+        """
```

Comment:
I think this is a style regression
```suggestion
        """Enable depth peeling."""
```

Reply:
Really? locally `make docstring` was complaining about it. I'll revert :+1: